base/applications/mspaint/dialogs.cpp

```diff
@@ -320,15 +320,15 @@ void CFontsDialog::InitFontSizes()
     TCHAR szText[16];
     for (UINT i = 0; i < _countof(s_sizes); ++i)
     {
-        wsprintf(szText, TEXT("%d"), s_sizes[i]);
+        StringCchPrintf(szText, _countof(szText), TEXT("%d"), s_sizes[i]);
         INT iItem = ComboBox_AddString(hwndSizes, szText);
         if (s_sizes[i] == (INT)registrySettings.PointSize)
             ComboBox_SetCurSel(hwndSizes, iItem);
     }
 
     if (ComboBox_GetCurSel(hwndSizes) == CB_ERR)
     {
-        wsprintf(szText, TEXT("%d"), (INT)registrySettings.PointSize);
+        StringCchPrintf(szText, _countof(szText), TEXT("%d"), (INT)registrySettings.PointSize);
```

Comment:
I would suggest you do that in a very separate commit, outside of this PR.

Reply:
Without these changes, the build will fail.

Reply:
ah yes, the mspaint would need an explicit `#include <strsafe.h>` in its precompiled header.

Reply:
OK now, you can move both this change and the one in the mspaint's precompiler header into a separate commit, I guess... ? 😄

Reply:
Reverted.